defs/pandaanalysis.def

```diff
@@ -0,0 +1,4 @@
+<require panda>
+
+{EventAnalysis>Event}
+genParticlesU/UnpackedGenParticleCollection(256)
```

Comment:
We need
`genParticlesU.parent->genParticlesU`
to get the parent ref working.

Reply:
Ah right thanks. I wasn't using parentage, but I'll add it for completeness.